src/SUMMARY.md

```diff
@@ -37,6 +37,7 @@
 - [Implementing new features](./implementing_new_features.md)
 - [Stability attributes](./stability.md)
 - [Stabilizing Features](./stabilization_guide.md)
+- [Using git](./git.md)
```

Comment:
This could go higher in the list I think, maybe just before 'Mastering @rustbot'.